GoNative sample linking error on Ubuntu

The GoNative app cannot be built using the newest attach-extended library. It appears to be a problem during the linking phase.


```
[Mo. Juni 27 09:32:17 MESZ 2022][INFORMATION] JavaFX static libs downloaded successfully
[Mo. Juni 27 09:32:18 MESZ 2022][INFORMATION] [SUB] Warning: Ignoring server-mode native-image argument --no-server.
[Mo. Juni 27 09:32:20 MESZ 2022][INFORMATION] [SUB] ========================================================================================================================
[Mo. Juni 27 09:32:20 MESZ 2022][INFORMATION] [SUB] GraalVM Native Image: Generating 'com.gluonhq.samples.gonative.gonative' (shared library)...
[Mo. Juni 27 09:32:20 MESZ 2022][INFORMATION] [SUB] ========================================================================================================================
[Mo. Juni 27 09:32:21 MESZ 2022][INFORMATION] [SUB] [1/7] Initializing...                                                                                    (2,3s @ 0,17GB)
[Mo. Juni 27 09:32:21 MESZ 2022][INFORMATION] [SUB]  Version info: 'GraalVM 22.1.0.1 Java 17 CE'
[Mo. Juni 27 09:32:21 MESZ 2022][INFORMATION] [SUB]  Garbage collector: Serial GC
[Mo. Juni 27 09:32:50 MESZ 2022][INFORMATION] [SUB] [2/7] Performing analysis...  [**********]                                                              (28,8s @ 2,58GB)
[Mo. Juni 27 09:32:50 MESZ 2022][INFORMATION] [SUB]   12.431 (90,90%) of 13.676 classes reachable
[Mo. Juni 27 09:32:50 MESZ 2022][INFORMATION] [SUB]   22.148 (65,87%) of 33.623 fields reachable
[Mo. Juni 27 09:32:50 MESZ 2022][INFORMATION] [SUB]   60.831 (64,85%) of 93.808 methods reachable
[Mo. Juni 27 09:32:50 MESZ 2022][INFORMATION] [SUB]      660 classes,   186 fields, and 2.155 methods registered for reflection
[Mo. Juni 27 09:32:50 MESZ 2022][INFORMATION] [SUB]      110 classes,   127 fields, and   183 methods registered for JNI access
[Mo. Juni 27 09:32:52 MESZ 2022][INFORMATION] [SUB] [3/7] Building universe...                                                                               (2,1s @ 3,40GB)
[Mo. Juni 27 09:32:54 MESZ 2022][INFORMATION] [SUB] [4/7] Parsing methods...      [*]                                                                        (1,8s @ 2,41GB)
[Mo. Juni 27 09:32:56 MESZ 2022][INFORMATION] [SUB] [5/7] Inlining methods...     [****]                                                                     (2,1s @ 4,01GB)
[Mo. Juni 27 09:33:17 MESZ 2022][INFORMATION] [SUB] [6/7] Compiling methods...    [****]                                                                    (20,9s @ 2,56GB)
[Mo. Juni 27 09:33:23 MESZ 2022][INFORMATION] [SUB] 
[Mo. Juni 27 09:33:23 MESZ 2022][INFORMATION] [SUB] ------------------------------------------------------------------------------------------------------------------------
[Mo. Juni 27 09:33:23 MESZ 2022][INFORMATION] [SUB]                         6,1s (9,4% of total time) in 38 GCs | Peak RSS: 6,93GB | CPU load: 6,40
[Mo. Juni 27 09:33:23 MESZ 2022][INFORMATION] [SUB] ------------------------------------------------------------------------------------------------------------------------
[Mo. Juni 27 09:33:23 MESZ 2022][INFORMATION] [SUB] Produced artifacts:
[Mo. Juni 27 09:33:23 MESZ 2022][INFORMATION] [SUB]  /home/michael/Workspaces/Experimental/Git/Gluon/gluon-samples/GoNative/target/gluonfx/aarch64-android/gvm/GoNative/com.gluonhq.samples.gonative.gonative.h (header)
[Mo. Juni 27 09:33:23 MESZ 2022][INFORMATION] [SUB]  /home/michael/Workspaces/Experimental/Git/Gluon/gluon-samples/GoNative/target/gluonfx/aarch64-android/gvm/GoNative/graal_isolate.h (header)
[Mo. Juni 27 09:33:23 MESZ 2022][INFORMATION] [SUB]  /home/michael/Workspaces/Experimental/Git/Gluon/gluon-samples/GoNative/target/gluonfx/aarch64-android/gvm/GoNative/com.gluonhq.samples.gonative.gonative_dynamic.h (header)
[Mo. Juni 27 09:33:23 MESZ 2022][INFORMATION] [SUB]  /home/michael/Workspaces/Experimental/Git/Gluon/gluon-samples/GoNative/target/gluonfx/aarch64-android/gvm/GoNative/graal_isolate_dynamic.h (header)
[Mo. Juni 27 09:33:23 MESZ 2022][INFORMATION] [SUB]  /home/michael/Workspaces/Experimental/Git/Gluon/gluon-samples/GoNative/target/gluonfx/aarch64-android/gvm/GoNative/com.gluonhq.samples.gonative.gonative.build_artifacts.txt
[Mo. Juni 27 09:33:23 MESZ 2022][INFORMATION] [SUB] ========================================================================================================================
[Mo. Juni 27 09:33:23 MESZ 2022][INFORMATION] [SUB] Finished generating 'com.gluonhq.samples.gonative.gonative' in 1m 4s.
[INFO] 
[INFO] --- gluonfx-maven-plugin:1.0.14:link (default-cli) @ gonative ---
[Mo. Juni 27 09:33:23 MESZ 2022][INFORMATION] ==================== LINK TASK ====================

Download progress: 0,01 / 0,02M
Download progress: 0,02 / 0,02M
Download progress: 0,02 / 0,02M
[Mo. Juni 27 09:33:24 MESZ 2022][INFORMATION] [SUB] /root/.gluon/substrate/Android/ndk-bundle/toolchains/llvm/prebuilt/linux-x86_64/bin/../lib/gcc/aarch64-linux-android/4.9.x/../../../../aarch64-linux-android/bin/ld.gold: error: /home/michael/Workspaces/Experimental/Git/Gluon/gluon-samples/GoNative/target/gluonfx/aarch64-android/gvm/lib/libatt_ext_log.a(log.o): multiple definition of 'JNI_OnLoad_log'
[Mo. Juni 27 09:33:24 MESZ 2022][INFORMATION] [SUB] /root/.gluon/substrate/Android/ndk-bundle/toolchains/llvm/prebuilt/linux-x86_64/bin/../lib/gcc/aarch64-linux-android/4.9.x/../../../../aarch64-linux-android/bin/ld.gold: /home/michael/Workspaces/Experimental/Git/Gluon/gluon-samples/GoNative/target/gluonfx/aarch64-android/gvm/lib/libattach_extended_log.a(log.o): previous definition here
[Mo. Juni 27 09:33:24 MESZ 2022][INFORMATION] [SUB] /root/.gluon/substrate/Android/ndk-bundle/toolchains/llvm/prebuilt/linux-x86_64/bin/../lib/gcc/aarch64-linux-android/4.9.x/../../../../aarch64-linux-android/bin/ld.gold: error: /home/michael/Workspaces/Experimental/Git/Gluon/gluon-samples/GoNative/target/gluonfx/aarch64-android/gvm/lib/libatt_ext_log.a(log.o): multiple definition of 'Java_com_gluonhq_attachextended_log_impl_AndroidLogService_logMessage'
[Mo. Juni 27 09:33:24 MESZ 2022][INFORMATION] [SUB] /root/.gluon/substrate/Android/ndk-bundle/toolchains/llvm/prebuilt/linux-x86_64/bin/../lib/gcc/aarch64-linux-android/4.9.x/../../../../aarch64-linux-android/bin/ld.gold: /home/michael/Workspaces/Experimental/Git/Gluon/gluon-samples/GoNative/target/gluonfx/aarch64-android/gvm/lib/libattach_extended_log.a(log.o): previous definition here
[Mo. Juni 27 09:33:25 MESZ 2022][INFORMATION] [SUB] clang++: error: linker command failed with exit code 1 (use -v to see invocation)
[Mo. Juni 27 09:33:25 MESZ 2022][SCHWERWIEGEND] Process link failed with result: 1
Check the log files under /home/michael/Workspaces/Experimental/Git/Gluon/gluon-samples/GoNative/target/gluonfx/aarch64-android/gvm/log
And please check https://docs.gluonhq.com/ for more information.
[Mo. Juni 27 09:33:25 MESZ 2022][INFORMATION] Logging process [link] to file: /home/michael/Workspaces/Experimental/Git/Gluon/gluon-samples/GoNative/target/gluonfx/log/process-link-1656315205426.log
[Mo. Juni 27 09:33:25 MESZ 2022][SCHWERWIEGEND] Linking failed.
Check the log files under /home/michael/Workspaces/Experimental/Git/Gluon/gluon-samples/GoNative/target/gluonfx/aarch64-android/gvm/log
And please check https://docs.gluonhq.com/ for more information.
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  01:15 min
[INFO] Finished at: 2022-06-27T09:33:25+02:00
[INFO] ------------------------------------------------------------------------
```

--- a/GoNative/pom.xml
+++ b/GoNative/pom.xml
@@ -17,7 +17,7 @@
 
     <properties>
         <main.class>com.gluonhq.samples.gonative.GoNative</main.class>
-        <attach.extended.version>4.0.13</attach.extended.version>
+        <attach.extended.version>4.0.14</attach.extended.version>
         <attach.extended.classifier>desktop</attach.extended.classifier>
     </properties>
 

